### DISCUSSION TITLE
USTL_ACM/

### DISCUSSION THREAD

**Discussion by giscus** [cargo-youth/ToolDiy#88] [Announcements]
# USTL_ACM/

https://risingrode.github.io/USTL_ACM/

**Comment by Risingrode** [comment]
如果想要贡献算法啥的，请联系本校acm队长，或者github提交issue。

**Comment by NorthSecond** [comment]
emmmmmmmmm，那请问这个项目和 [oi-wili](https://oi-wiki.org) 有什么区别吗

**Reply by Risingrode** [reply]
你好，跟oi-wiki并没有什么区别，可以说oi-wiki比这个好。不过目前需要的是符合纯新手小白可以接触的算法编程知识库，这个是面向我的学校的，致力于打造符合本校学生的算法平台。
